sdk/v1/client.go

```diff
@@ -81,6 +81,31 @@ func (cli *Client) PostJob(ctx context.Context, headers client.Headers) (models.
 	return job, nil
 }
 
+// PostTasksCount Updates tasks count for processing
+func (cli *Client) PostTasksCount(ctx context.Context, headers client.Headers, jobID string) (models.Task, error) {
+	var task models.Task
+
+	if headers.ServiceAuthToken == "" {
+		headers.ServiceAuthToken = cli.serviceToken
+	}
+
+	path := fmt.Sprintf("%s/jobs/%s/tasks", cli.apiVersion, jobID)
+
+	b, err := cli.callReindexAPI(ctx, path, http.MethodPost, headers, nil)
```

Comment:
The body should not be nil. It should be the details of the task to create/update, which is defined in the models package by:
 type TaskToCreate struct {
	TaskName          string `json:"task_name"`
	NumberOfDocuments int    `json:"number_of_documents"`
}

Reply:
well spotted....now implemented